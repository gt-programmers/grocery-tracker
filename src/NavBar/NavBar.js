import React from "react";
import "./NavBar.scss";
import { NavLink } from "react-router-dom";

const NavBar = () => {
  return (
    <>
      <nav>
        <NavLink to={"/profile"} className="route" >Profile</NavLink>
        <NavLink to={"/grocery_list"} className="route" >Grocery List</NavLink>
        <NavLink to={"/inventory"} className="route" >Inventory</NavLink>
        <NavLink to={"/recipes"} className="route" >Recipes</NavLink>
        <NavLink to={"/stats"} className="route" >Stats</NavLink>
      </nav>
    </>
  );
};

export default NavBar;
