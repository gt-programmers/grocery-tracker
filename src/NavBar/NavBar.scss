nav {
  display: flex;
  flex-direction: row; 
  justify-content: space-around;
  background-color: #18bf18;
  
  .route {
    text-decoration: none;
    color: #ffffff;
    font-weight: bold;
    font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
    cursor: pointer;
    padding-top: 2vh;
    padding-bottom: 2vh;
  }
}