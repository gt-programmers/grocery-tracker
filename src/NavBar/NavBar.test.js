import React from 'react';
import NavBar from './NavBar';
import { shallow } from 'enzyme';

describe("NavBar", () => {
  let wrapper = shallow( <NavBar />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})


