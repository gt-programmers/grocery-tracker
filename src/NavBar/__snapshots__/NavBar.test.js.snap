// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the snapshot 1`] = `
<Fragment>
  <nav>
    <div
      className="vertical point"
    >
      <img
        alt="statistics"
        src="user.svg"
      />
      <NavLink
        className="route"
        to="/profile"
      >
        Profile
      </NavLink>
    </div>
    <div
      className="vertical point"
    >
      <img
        alt="statistics"
        src="groceries.svg"
      />
      <NavLink
        className="route"
        to="/grocery_list"
      >
        Grocery List
      </NavLink>
    </div>
    <div
      className="vertical point"
    >
      <img
        alt="statistics"
        src="clipboard.svg"
      />
      <NavLink
        className="route"
        to="/inventory"
      >
        Inventory
      </NavLink>
    </div>
    <div
      className="vertical point"
    >
      <img
        alt="statistics"
        src="chef.svg"
      />
      <NavLink
        className="route"
        to="/recipes"
      >
        Recipes
      </NavLink>
    </div>
    <div
      className="vertical point"
    >
      <img
        alt="statistics"
        src="bar-chart.svg"
      />
      <NavLink
        className="route"
        to="/stats"
      >
        Stats
      </NavLink>
    </div>
  </nav>
</Fragment>
`;
